### ISSUE TITLE
`angular` syntax profile from emmet doesn't work after 1.5.0 update

### ISSUE THREAD

**Issue by AldoMX** [Microsoft/vscode#11799]
- VSCode Version: 1.5.1
- OS Version: Microsoft Windows [Version 6.1.7601]

Steps to Reproduce:

1 - Set the `angular` syntax profile in your `settings.json`, example:
```
{
    "emmet.syntaxProfiles": {
        "html": "angular"
    }
}
```
2 - Create a new HTML File

3 - Try to expand an emmet expression, example:
```
ul>li*5
```

3b - Or an angular-specific emmet expression, example:
```
md-content[layout="column"]>md-button{Save}
```

I noticed that both versions of VS Code: 1.4.0 and 1.5.1 have the same version of emmet installed (1.3.1).

**Comment by mrmlnc** [comment]
Hello, @AldoMX,

What do you mean by `angular` profile?

**Comment by AldoMX** [comment]
I think I'm getting crazy...

Apparently the `angular` profile never existed although I was pretty confident that it did, and I took it for granted since I had found out undocumented emmet's features in the past, such as the `|slim` or `|jade` filters.

Anyway, the "issue" is real, but it will probably be a WONTFIX:

In VS Code <= 1.4.0, when you press tab and the emmet profile is invalid, ex.:
```
{
    "emmet.syntaxProfiles": {
        "html": "gibberish"
    }
}
```

It triggers emmet anyway, which results in emmet expanding invalid expressions (but valid angular markup) such as
```
ng-app
```
into
```
<ng-app></ng-app>
```

Which led me to believe the non-existent `angular` profile actually existed.

Basically, VS Code 1.5.0 fixed the bug, broke my workflow, and made me learn that I was probably asleep when I found out the `angular` profile...

Please accept an apology for this bogus bug report.

**Comment by mrmlnc** [comment]
Don't worry. Now `syntaxProfiles` uses as `fileAssociations`, but for Emmet: left side — language Id, right side — Emmet mode. You can find the basic modes of Emmet in https://github.com/Microsoft/vscode/issues/9500#issuecomment-242221652.

Please, close this issue if it is resolved.

**Comment by ramya-rao-a** [comment]
@AldoMX Emmet abbreviations should work for you in html files without setting any syntax profiles.
Closing this issue as resolved.
Love the xkcd comic :)
